src/util/prometheus.js

```diff
@@ -614,18 +614,6 @@ class Prometheus {
           'k8_namespace',
         ],
       },
-      {
-        name: 'user_transform_function_latency',
```

Comment:
https://github.com/rudderlabs/rudder-transformer/blob/e7dbd2cc2b040683ca65356bf9d4258b748aaa2c/src/legacy/router.js#L631 this is referred here so it might cause issue if someone is using legacy routes.

Reply:
It's legacy router and not used anymore. We plan to remove this file eventually.

Reply:
what about vpc customers? may be we won't support UTs for vpc customers right?